sos/report/plugins/fibrechannel.py

```diff
@@ -18,8 +18,20 @@ class Fibrechannel(Plugin, RedHatPlugin):
     plugin_name = 'fibrechannel'
     profiles = ('hardware', 'storage', 'system')
     files = ('/sys/class/fc_host', '/sys/class/fc_remote_ports')
+    option_list = [("debug", "enable debug logs", "fast", True),
+                   ("add_dbg_path", "add debug path to collect", "fast", None)
```

Comment:
So we still end up with the same end issue here - user defined paths. All it would take is a user typo here to land us in a lot of hot water (imagine a user does `-k fibrechannel.add_dbg_path=/` for example).

What I am saying for this is to not allow user defined paths, and simply maintain a list of well known vendor paths for this kind of information.

Reply:
Well, I agree with you. I wanted to have an easy way to add paths for collecting specific logs in future without pull requests. You're right it may lead to negative effects if someone uses this mindlessly. So I've removed it.